app/src/main/res/layout-v21/login.xml

```diff
@@ -140,6 +140,18 @@
             android:visibility="gone" />
 
         <TextView
+            android:id="@+id/register"
```

Comment:
Nice! How about improving it further by using styles to club common properties of Another server, Register and legal info TextViews?